### ISSUE TITLE
[Bug] Filter out closed auctions from My Account > Dashboard

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#666]
THIS IS A FUN BUG! To test this locally, you'll need to a few Nonprofit Sites set up. Then, you'll: 

- Create and publish an Auction for Nonprofit Site A that's ending soon 
- Bid in the Auction 
- Wait until it closes 
- Check out My Account > Dashboard for Nonprofit Site A, confirm the closed Auction does not show in the "GET BACK IN THE ACTION" section 
- Navigate to Nonprofit Site B, go to My Account > Dashboard, see the closed Auction on the page. (Link goes to "page not found" page.) 
- Navigate to the GOODBIDS Main Site, go to My Account > Dashboard, see the closed Auction on the page. (Link goes to "page not found" page.) 

Basically, it looks like closed Auctions are getting filtered out of My Account > Dashboard on the Nonprofit Site where the Auction took place, but on other sites, the Auction still displays.
